leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -228,14 +227,10 @@ var LinkInspector = Backbone.View.extend({
             $("#repeat-error").text("");
         } else if(begin == "and" || begin == "or") {
             this.setSelectValues('#link-type-end', 'A');
-
-            var end = $("#link-type-end").val();
-
             $("#link-type-end").prop('disabled', false);
             $("#link-type-end").css("background-color","");
             
             //Saving this option
-            this.cell.prop("link-type", begin + "|" + end);
```

Comment:
Why did you delete this line? Asking just because I don't know as much about link-type properties as you do, and just want to make sure we are still saving the type of link into the cell somewhere

Reply:
When I commented it out, it didn't make a difference because we have another line after it that does the same thing, so I deleted it because it is repetitive.

Reply:
I looked but couldn't find the line you are referring to that does the same thing - could you direct me to it (just send me the number of the line in your branch)? Thanks!

Reply:
line 261, in updateEndEvolRelations. It is also important to note that the command I deleted was only in the AND/OR if statement, the other if statements did not include it, even the ones that have a begin and an end.

Reply:
I see that, but as far as I can tell updateEndEvolRelations is not always called after updateBeginEvolRelations. For example, if someone changes the beginEvolRelationship, now that will not update the link-type property. Whether or not we need it to I think is a question we can work through as linkInspector.js is implemented with backbone, but in the meantime I think deleting this line is out of scope of the pull request so I would add it back in temporarily until we can look into it more